s2fft/precompute_transforms/construct.py

```diff
@@ -193,39 +369,103 @@ def wigner_kernel(
         forward (bool, optional): Whether to provide forward or inverse shift.
             Defaults to False.
 
+        mode (str, optional): Whether to use FFT approach or manually compute each element.
+            {"auto", "manual", "irfft"}. Defaults to "auto" which will detect the
```

Comment:
The docstring says "irfft" but it seems "fft" is used in the code?  "fft" as used in the code is probably better since it less technical.  

I'm surpised tests haven't caught this.  Can you also add a test that would catch this?

Reply:
This is just a docstring difference so there isn't a way to test this. The code is otherwise consistent, and I've now updated the docstring to "fft" to keep things simple.